framework/codemodder-testutils/src/main/java/io/codemodder/testutils/CodemodTestMixin.java

```diff
@@ -111,8 +114,21 @@ private void verifyCodemod(
       pathToJavaFile = newPathToJavaFile;
     }
 
-    // add the sonar JSON if one exists
-    Path sonarJson = testResourceDir.resolve("sonar-issues.json");
+    final List<String> sonarJsons =
+        sonarIssuesJsonFiles != null ? Arrays.asList(sonarIssuesJsonFiles) : new ArrayList<>();
+
+    final List<Path> sonarJsonsPaths =
+        sonarJsons.stream()
+            .map(testResourceDir::resolve)
+            .filter(Files::exists)
+            .collect(Collectors.toList());
+
+    if (sonarJsonsPaths.isEmpty()) {
+      Path defaultPath = testResourceDir.resolve("sonar-issues.json");
```

Comment:
Sorry, I thought I left this comment before but I don't see it. Shouldn't it be this now?
```suggestion
      Path defaultPath = testResourceDir.resolve("sonar-issues_1.json");
```

Reply:
@drdavella no, default sonar issues filename remains the same as it previously was, otherwise, i will have to rename all other sonar codemod test json files 
<img width="469" alt="Screenshot 2024-05-13 at 12 51 17 p m" src="https://github.com/pixee/codemodder-java/assets/14047435/1c529923-233d-45bb-a1f3-a41184a6db86">

Reply:
Oh okay, I'm sorry. I missed that the one you renamed was for a specific test.

Reply:
yes